### PR TITLE
5802 Fix GetAreas 'totalCount' and 'count'

### PR DESCRIPTION
**Pull request by PauloPortugal**
### What

Resolves: [5802](https://trello.com/c/7MD0oUPR/5802-bug-fix-count-on-review-page)

### How to review

Sense check it and tests are green

### Who can review

Any ONS developer

### CODE REVIEW

File: cantabular/queries.go [hunk 1]
```diff
@@ -208,6 +208,7 @@ query ($dataset: String!, $text: String!, $category: String!, $limit: Int!, $off
 			name
 			label
 			categories {
+              totalCount
```
Comment (j-s-rawat) [inline cantabular/queries.go:211]: not a issue but we might indent it

**Review by j-s-rawat — DISMISSED** [review]
LGTM

File: cantabular/queries.go (hunk 1 above)
Comment (PauloPortugal) [reply cantabular/queries.go:211]: oh yes...

Comment (PauloPortugal) [reply cantabular/queries.go]: this has been addressed

**Review by j-s-rawat — APPROVED** [review]
LGTM

**Pull request merged**